### ISSUE TITLE
Elevated GFAP (Glial fibrillary acidic protein) level in CSF

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5149]
https://clinicaltrials.gov/ct2/show/NCT02998879
https://www.ncbi.nlm.nih.gov/pubmed/23494656
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4693626

**Comment by pnrobinson** [comment]
@Marwaabdelhakim

**Comment by pnrobinson** [comment]
Label: Elevated CSF glial fibrillary acidic protein level
parent - https://hpo.jax.org/app/browse/term/HP:0025456
Definition: Increased concentration of glial fibrillary acidic protein n cerebrospinal fluid.
Comment: Elevated CSF glial fibrillary acidic protein levels are considered to be a glial marker of neuroinflammation

PMID:31892365


Also add analogous terms for chitotriosidase 1 (CHIT1) and chitinase-3-like protein 1 (YKL-40) 
https://www.ncbi.nlm.nih.gov/pubmed/31892365/
